### PR TITLE
New package: kotlin-1.2.51

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/kotlin/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Template file for 'kotlin'
+pkgname=kotlin
+version=1.2.51
+revision=1
+wrksrc="${pkgname}c"
+only_for_archs="i386 x86_64"
```
Comment (maxice8) [inline srcpkgs/kotlin/template]: i686 ?

File: srcpkgs/kotlin/template [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+makedepends="unzip setconf"
+depends="openjdk"
+short_desc="Statically typed programming language that can interoperate with Java"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="Apache"
```
Comment (maxice8) [inline srcpkgs/kotlin/template]: please use spdx identifier, should be Apache-2.0

File: srcpkgs/kotlin/template (hunk 1 above)
Comment (shizonic) [reply srcpkgs/kotlin/template]: You are right. It's only x86_64. Thanks.

File: srcpkgs/kotlin/template (hunk 2 above)
Comment (shizonic) [reply srcpkgs/kotlin/template]: Alright. Thanks!

**Comment by Duncaen** [comment]
@cr6git I don't think this was ready to merge.

**Comment by Duncaen** [comment]
x86_64 only? Maybe build it from source (not sure about any policy about this for java), but we usually add a `-bin` suffix to packages that are not built form source.

**Comment by shizonic** [comment]
@Duncaen I see. Anything else?

**Comment by Duncaen** [comment]
I don't like the use of `setconf` in templates. Useless pattern replacement `${version/_/-}` in `distfiles`.

**Comment by shizonic** [comment]
@Duncaen Could you help me to change the template without setconf ?

**Comment by shizonic** [comment]
@Duncaen How can I withdraw the already happened merge?

**Comment by cr6git** [comment]
@Duncaen ok, I probably merged too fast. This template was based on [this PKGBUILD](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/kotlin), I thought it was not bad... well, I apologize.

**Comment by shizonic** [comment]
@cr6git & @Duncaen Correct. I adapted it from arch's PKGBUILD. I just made a new PR (https://github.com/void-linux/void-packages/pull/1220) where I even renamed the package to `-bin`. Hope I've done this the correct way.

And I am open for further improvements but would need some help.

**Pull request merged**